worlds/cccharles/docs/en_Choo-Choo Charles.md

```diff
@@ -0,0 +1,39 @@
+# Choo-Choo Charles
+
+## Game page in other languages
+* [Français](fr)
+
+## Where is the options page?
+The [Player Options page](../player-options) contains all the options to configure and export a yaml config file.
+
+## What does randomization do to this game?
+All scraps or any collectable item on the ground (except from Loot Crates) and items received from NPCs missions are considered as locations to check.
+
+## What is the goal of Choo-Choo Charles when randomized?
+Beating the evil train from Hell named "Charles".
+
+## How is the game managed in Nightmare mode?
+At death, the player has to restart a brand-new game, giving him the choice to stay under the Nightmare mode or continuing with the Normal mode if considered too hard.
+In this case, all collected items will be redistributed in the inventory and the missions states will be kept.
+The Deathlink is not implemented yet. When this option will be available, a choice will be provided to :
```

Comment:
```suggestion
The Deathlink is not implemented yet. When this option will be available, a choice will be provided to:
```
And also some spaces before user-facing colons. Up to you if it's preferred I suppose.